### PR TITLE
Security fix

### PR DESCRIPTION
**Pull request by RDIL**
Somewhere down in the dependency tree, this update fixes an issue (via `react-ace` update).
Also update `react-markdown` because it fixed a bug and bugs are big baddies.

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ container:
   image: node:12
 
 build_task:
+  auto_cancellation: false
```
Comment (fkorotkov) [inline .cirrus.yml]: Why dependabot is confused when tasks are aborted?

Comment (RDIL) [reply .cirrus.yml]: It’s actually Cirrus, it commits and then opens the PR, and without this it gets confused and cancels both runs 🤷‍♂

Comment (RDIL) [reply .cirrus.yml]: (The timing is very close together so I think cirrus has trouble deciding which to cancel)

Comment (fkorotkov) [reply .cirrus.yml]: This is a workaround and not a fix. Please create a separate issue describing what's happening with dependabot PRs so we can think of an actual solution.

Comment (RDIL) [reply .cirrus.yml]: Okay

**Pull request merged**